src/dotnet/commands/dotnet-test/Program.cs

```diff
@@ -70,6 +69,12 @@ public static TestCommand FromArgs(string[] args, string msbuildPath = null)
                 }
             }
 
+            // Add VSTestNoLogo property if specified by user
+            if(parsedTest.Arguments.Contains("--nologo") || parsedTest.Arguments.Contains("/nologo"))
```

Comment:
@livarcocc @peterhuene Are dotnet cli args case sensitive?

Reply:
I think we don't need `/nologo` here.

Reply:
in the bug user is passing /nologo, & the argument comes as /nologo

Reply:
@smadala they are if using the command parser; have we considered making this an actual option to the `test` command (i.e. change `TestCommandParser`)?

Reply:
@peterhuene making change to TestCommandParser, would mean we are supporting it as first class argument, which we aren't in vstest.console.exe

Reply:
Not a fan of magical MSBuild properties keyed off of MSBuild switches, but that's fine.